src/types/inventory/stackrequest/CraftRecipeAutoStackRequestAction.php

```diff
@@ -34,13 +34,16 @@ final class CraftRecipeAutoStackRequestAction extends ItemStackRequestAction{
 	 */
 	final public function __construct(
 		private int $recipeId,
+		private int $repetitions2,
 		private int $repetitions,
 		private array $ingredients
 	){}
 
 	public function getRecipeId() : int{ return $this->recipeId; }
 
-	public function getRepetitions() : int{ return $this->repetitions; }
+	public function getRepetitions() : int { return $this->repetitions; }
```

Comment:
remove space for code consistency
```suggestion
	public function getRepetitions() : int{ return $this->repetitions; }
```